.eslintrc.yml

```diff
@@ -9,22 +9,28 @@ env:
   node: true
 
 rules:
+  arrow-body-style: 2
+  arrow-parens: [2, as-needed]
```

Comment:
I personally would prefer `as-needed` over `always`, but I won't fight over it :)

I completely get what you're saying, but there is a small subtlety to it though:

```js
if (something)
  doThis();
  doThat();
```

The above is perfectly valid JS, but probably not what one wants. However:

```js
arr.map(x, y => x * 2); // 💥
```

is simply broken JS, and a typo that one of our checks (tests, linters, compilers) should detect, regardless of the style preference.

Reason why I'd rather keep `as-needed` is because of the beauty/simplicty of the new syntax: 

```js
arr.map(x => x * 2);
```

This is much nicer than:

```js
arr.map((x) => x * 2);
```

And the reason why I won't fight over it is because parentheses are needed in other scenarios:

```js
myFunction(() => doSomething());
myOtherFunction((x, y) => doSomethingElse(x, y));
```

And also because there is more important in life 😅
Anyway, my recommendation is to keep `as-needed` and if we start seeing some issues with it, we can become more strict about it.